### PR TITLE
New test terraform file

### PR DESCRIPTION
**Pull request by yane3628**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: test3.tf [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+resource "azurerm_sql_firewall_rule" "example" {
+  name                = "firewall-rule"
+  resource_group_name = "testrg"
+  server_name         = azurerm_sql_server.example.name
+  start_ip_address    = "10.0.17.62"
+  end_ip_address      = "10.0.17.62"
+}
+
+resource "azurerm_storage_account" "example" {
```
Comment (burn-er) [inline test3.tf:34]: :exclamation:**Cycode**: Infrastructure configuration issue: 'Ensure default network access rule for Storage Accounts is set to deny'.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |
| **#cycode_iac_fix_this_violation** | Fix this violation via a commit to this branch |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |

Comment (burn-er) [inline test3.tf:34]: :exclamation:**Cycode**: Infrastructure configuration issue: 'Ensure Storage Account is using the latest version of TLS encryption'.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |

File: test3.tf [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  administrator_login_password = "Aa12345678"
+  version                      = "9.5"
+  ssl_enforcement_enabled      = true
+}
+
+resource "azurerm_postgresql_configuration" "thrtottling_config" {
+  name                = "connection_throttling"
+  resource_group_name = "testrg"
+  server_name         = azurerm_postgresql_server.example.name
+  value               = "off"
```
Comment (burn-er) [inline test3.tf:123]: :exclamation:**Cycode**: Infrastructure configuration issue: 'Ensure server parameter 'connection_throttling' is set to 'ON' for PostgreSQL Database Server'.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |
| **#cycode_iac_fix_this_violation** | Fix this violation via a commit to this branch |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |

File: test3.tf [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+resource "azurerm_postgresql_configuration" "example" {
+  name                = "log_checkpoints"
+  resource_group_name = "testrg"
+  server_name         = azurerm_postgresql_server.example.name
+  value               = "off"
```
Comment (burn-er) [inline test3.tf:130]: :exclamation:**Cycode**: Infrastructure configuration issue: 'Ensure server parameter 'log_checkpoints' is set to 'ON' for PostgreSQL Databa'.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |
| **#cycode_iac_fix_this_violation** | Fix this violation via a commit to this branch |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |

File: test3.tf [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+  sku_name   = "GP_Gen5_2"
+  storage_mb = 5120
+  version    = "5.7"
+
+  auto_grow_enabled                 = true
+  backup_retention_days             = 7
+  infrastructure_encryption_enabled = true
+  public_network_access_enabled     = true
+  ssl_enforcement_enabled           = false
```
Comment (burn-er) [inline test3.tf:101]: :exclamation:**Cycode**: Infrastructure configuration issue: 'Ensure 'Enforce SSL connection' is set to 'ENABLED' for MySQL Database Server'.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |
| **#cycode_iac_fix_this_violation** | Fix this violation via a commit to this branch |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |

File: test3.tf [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+    minute_metrics {
+      enabled               = true
+      include_apis          = true
+      version               = "1.0"
+      retention_policy_days = 10
+    }
+  }
+}
+
+resource "azurerm_sql_server" "example" {
```
Comment (burn-er) [inline test3.tf:63]: :exclamation:**Cycode**: Infrastructure configuration issue: 'Ensure that 'Auditing' is set to 'On''.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |

File: test3.tf (hunk 1 above)
Comment (burn-er) [inline test3.tf:34]: :exclamation:**Cycode**: Infrastructure configuration issue: 'Ensure that 'Secure transfer required' is set to 'Enabled''.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |
| **#cycode_iac_fix_this_violation** | Fix this violation via a commit to this branch |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |

File: test3.tf [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+resource "azurerm_postgresql_server" "example" {
+  name                         = "postgresql-${random_integer.rnd_int.result}"
+  location                     = "East US"
+  resource_group_name          = "testrg"
+  sku_name                     = "B_Gen5_2"
+  storage_mb                   = 5120
+  backup_retention_days        = 7
+  geo_redundant_backup_enabled = false
```
Comment (burn-er) [inline test3.tf:111]: :exclamation:**Cycode**: Infrastructure configuration issue: 'Enable PostgreSQL geo-redundant backups'.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |
| **#cycode_iac_fix_this_violation** | Fix this violation via a commit to this branch |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |

File: test3.tf (hunk 1 above)
Comment (burn-er) [inline test3.tf:34]: :exclamation:**Cycode**: Infrastructure configuration issue: 'Ensure 'Trusted Microsoft Services' is enabled for Storage Account access'.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |

File: test3.tf (hunk 5 above)
Comment (burn-er) [inline test3.tf:63]: :exclamation:**Cycode**: Infrastructure configuration issue: 'Ensure that Azure Active Directory Admin is configured'.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |

File: test3.tf (hunk 1 above)
Comment (yane3628) [reply test3.tf:34]: #cycode_iac_fix_this_violation

Comment (burn-er) [reply test3.tf:34]: :-1: Cycode does not support this action for this violation at this time.

**Pull request merged**